desktop/package.json

```diff
@@ -49,19 +50,25 @@
     "browser-sync": "^2.10.0",
     "electron-compile": "^1.0.1",
     "electron-packager": "^5.1.0",
-    "electron-prebuilt": "^0.35.4",
-    "electron-rebuild": "^1.0.0",
+    "electron-prebuilt": "mafintosh/electron-prebuilt#d76ae84fcd974114aa4da2809b8442ad8b9d43c2",
```

Comment:
electron prebuilt has an issue with electron rebuild, this is a snapshot of master which fixes that issue